mabwiser/base_mab.py

```diff
@@ -94,12 +116,14 @@ def add_arm(self, arm: Arm, binarizer: Callable = None) -> NoReturn:
         """
         self.arm_to_expectation[arm] = 0
         self._uptake_new_arm(arm, binarizer)
+        self.arm_to_status[arm] = {STATUS_TRAINED: False, STATUS_WARM: False, STATUS_WARM_STARTED_BY: None}
```

Comment:
why not call these IS_TRAINED, IS_WARM, WARM_STARTED_BY? 
May be I am missing sth

Reply:
I'll update